src/Features/CSharp/Portable/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -18,6 +18,20 @@ public static bool IsFunctionDeclaration(SyntaxNode node)
             || node is LocalFunctionStatementSyntax
             || node is AnonymousFunctionExpressionSyntax;
 
+        public static ImmutableArray<SyntaxNode> GetParameters(SyntaxNode functionDeclaration)
+        {
+            return functionDeclaration switch
+            {
+                null => default,
+                BaseMethodDeclarationSyntax methodDeclaration => methodDeclaration.ParameterList.Parameters.SelectAsArray(p => p as SyntaxNode),
+                LocalFunctionStatementSyntax localFunction => localFunction.ParameterList.Parameters.SelectAsArray(p => p as SyntaxNode),
+                AnonymousMethodExpressionSyntax anonymousMethodExpression => anonymousMethodExpression.ParameterList.Parameters.SelectAsArray(p => p as SyntaxNode),
+                SimpleLambdaExpressionSyntax simpleLambdaExpression => ImmutableArray.Create<SyntaxNode>(simpleLambdaExpression.Parameter),
+                ParenthesizedLambdaExpressionSyntax lambdaExpressionSyntax => lambdaExpressionSyntax.ParameterList.Parameters.SelectAsArray(p => p as SyntaxNode),
+                _ => throw ExceptionUtilities.UnexpectedValue(functionDeclaration),
+            };
+        }
```

Comment:
i don't this we need this.  we already have CSharpSyntaxGenerator.GetParameterList